### ISSUE TITLE
Resolution fails for second and subsequent builds when using Gradle daemon

### ISSUE THREAD

**Issue by mikewatt** [JFrogDev/build-info#99]
Example project that I am reproducing this with can be found here: https://github.com/mikewatt/gradle-artifactory-issue

Assuming no gradle daemon is running, running `./gradlew clean build` twice in succession from the root project directory results in:

First invocation:

``` shell
Starting a new Gradle Daemon for this build (subsequent builds will be faster).
[buildinfo] Not using buildInfo properties file for this build.
:sp1:clean UP-TO-DATE
:sp1:compileJava
:sp1:processResources UP-TO-DATE
:sp1:classes
:sp1:jar
:sp1:assemble
:sp1:compileTestJava UP-TO-DATE
:sp1:processTestResources UP-TO-DATE
:sp1:testClasses UP-TO-DATE
:sp1:test UP-TO-DATE
:sp1:check UP-TO-DATE
:sp1:build

BUILD SUCCESSFUL

Total time: 6.309 secs
```

Second invocation:

``` shell
:sp1:clean
:sp1:compileJava

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':sp1:compileClasspath'.
> Cannot resolve external dependency com.google.guava:guava:19.0 because no repositories are defined.
  Required by:
      com.example:sp1:1.0-SNAPSHOT

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

After killing the Gradle daemon spawned in the first invocation, I can then build successfully again (once).

The Artifactory service is an Artifactory Online instance. I'm running this on OS X 10.11.6 with java 1.8.0_60.

This seems to be a new issue with the 4.4.+ version of the plugin, as I cannot reproduce it with 4.3.0.

**Comment by eyalbe4** [comment]
Thanks @mikewatt. 
Looks like this is a compatibility issue with Gradle 3.0
We are working on it and will update soon.

**Comment by eyalbe4** [comment]
@mikewatt,
We've jsu committed that fixes this issue and created a snapshot version of the plugin with the fix.
Will you be able to try out the snapshot and let us know if it resolves the issue?
You can install the snapshot as follows:
```
buildscript {
	repositories {	
		maven() {
			url 'https://oss.jfrog.org/artifactory/oss-snapshot-local'
		}        		
	}
	dependencies {
		classpath "org.jfrog.buildinfo:build-info-extractor-gradle:4.4.x-SNAPSHOT"
	}
}
```
Thanks.

**Comment by mikewatt** [comment]
@eyalbe4 
Yes, looks like it's fixed the issue. Thanks very much!

**Comment by lptr** [comment]
Looks to be solved with plugin version 4.4.9.

**Comment by eyalbe4** [comment]
Thanks for the feedback @lptr and @mikewatt.
I'm closing this issue.
